packages/framework/src/Resources/config/services_test.yaml

```diff
@@ -24,9 +24,9 @@ services:
     prezent_doctrine_translatable.listener:
         arguments:
             $factory: '@prezent_doctrine_translatable.metadata_factory'
-            $domain: '@Shopsys\FrameworkBundle\Component\Domain\Domain'
-            $administrationFacade: '@Shopsys\FrameworkBundle\Model\Administration\AdministrationFacade'
-            $adminLocale: '%shopsys.admin_locale%'
+        #            $domain: '@Shopsys\FrameworkBundle\Component\Domain\Domain'
+        #            $administrationFacade: '@Shopsys\FrameworkBundle\Model\Administration\AdministrationFacade'
+        #            $localization: '@Shopsys\FrameworkBundle\Model\Localization\Localization'
```

Comment:
should be this commented?

Reply:
ah, damn :smile: I was wondering whether these lines can be removed :smile: (yes, they can be removed, I will add a fixup)